src/api/cloudDb/enterprise/backup/backup.iceberg.service.js

```diff
@@ -0,0 +1,12 @@
+angular.module('ovh-api-services').service('OvhApiCloudDBEnterpriseBackupIceberg', (iceberg) => {
+  const backupResource = iceberg('/cloudDB/enterprise/cluster/:clusterId/backup/:backupId', {
+    clusterId: '@clusterId',
+    backupId: '@backupId',
+  }, {
+    post: { method: 'POST' },
+    put: { method: 'PUT' },
+    get: { method: 'GET' },
```

Comment:
You have default functions exposed for this, you don't need to override as you don't add extra config to default methods

Reply:
These have been removed